CHANGELOG.unreleased.md

```diff
@@ -13,6 +13,9 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [Proxy] Fix proxy deploy timeout in case of not correctly get the proxy VM IP [#4927](https://github.com/vatesfr/xen-orchestra/issues/4927) (PR [#4928](https://github.com/vatesfr/xen-orchestra/pull/4928))
+- [Proxy] Fix "cannot get the proxy VM IP" error in case of not correctly get the proxy VM IP [#4927](https://github.com/vatesfr/xen-orchestra/issues/4927) (PR [#4928](https://github.com/vatesfr/xen-orchestra/pull/4928))
```

Comment:
This sentence is unclear and brings no clear info.

Reply:
`Fix non-pertinent throw of the error "cannot get the proxy VM IP".` ?